static/app/views/automations/components/actionFilters/eventFrequency.tsx

```diff
@@ -1,16 +1,31 @@
+import {Flex} from 'sentry/components/container/flex';
+import {RowLine} from 'sentry/components/workflowEngine/form/automationBuilderRowLine';
 import AutomationBuilderSelectField from 'sentry/components/workflowEngine/form/automationBuilderSelectField';
-import {tct} from 'sentry/locale';
+import {TextBadge} from 'sentry/components/workflowEngine/ui/textBadge';
+import {t, tct} from 'sentry/locale';
 import {DataConditionType} from 'sentry/types/workflowEngine/dataConditions';
 import {
   CountBranch,
   PercentBranch,
 } from 'sentry/views/automations/components/actionFilters/comparisonBranches';
+import {SubfiltersList} from 'sentry/views/automations/components/actionFilters/subfiltersList';
 import {useDataConditionNodeContext} from 'sentry/views/automations/components/dataConditionNodes';
 
 export default function EventFrequencyNode() {
-  return tct('Number of events in an issue is [select]', {
-    select: <ComparisonTypeField />,
-  });
+  const {condition} = useDataConditionNodeContext();
+  const hasSubfilters = condition.comparison.filters?.length > 0;
+
+  return (
+    <Flex column>
+      <RowLine>
+        {tct('Number of events in an issue is [select] [where]', {
+          select: <ComparisonTypeField />,
+          where: hasSubfilters && <TextBadge>{t('Where')}</TextBadge>,
+        })}
```

Comment:
where should always have a value might want to use a ternary and return null or not use tct to handle layout